Watchman.Discord/Areas/Protection/Services/UserMessagesCountService.cs

```diff
@@ -21,24 +22,45 @@ public UserMessagesCountService(IQueryBus queryBus)
 
         public int CountMessages(Contexts contexts)
         {
-            if (_serverMessagesQuantity == null || _lastUpdated.AddHours(12) < DateTime.UtcNow)
+            if (ShouldReloadCache())
             {
-                _lastUpdated = DateTime.UtcNow;
+                ReloadCache();
+            }
+
+            return GetOneUserFromOneServerCount(contexts.User.Id, contexts.Server.Id);
```

Comment:
potrzebujemy coś więcej z Contexts niż id użytkownika i id serwera?
myśle że bardziej uniwersalne to będzie jeśli będziemy przekazywać same ID 
albo osobny obiekt który ma tylko niezbędne dane 

chcąc użyć tej klasy w innym przypadku musielibyśmy generować contexts od nowa